internal/exec/stages/files/filesystemEntries.go

```
@@ -140,20 +140,34 @@ func (s *stage) createResultFile() error {
 	}
 
 	s.Logger.PushPrefix("createResultFile")
+	var prevReport interface{}
+	prevPath, _ := s.JoinPath(distro.ResultFilePath())
```

Why are we ignoring the error return here?